src/usage/context/usage.tsx

```diff
@@ -78,6 +78,17 @@ export const UsageContext = React.createContext<UsageContextType>(
   DEFAULT_CONTEXT
 )
 
+export const calculateCreditDaysUsed = (creditStartDate: string): number => {
+  const MILLISECONDS_IN_ONE_DAY = 1000 * 60 * 60 * 24
```

Comment:
It's not essential for this issue, but I think it's a good idea to move this constant to the [datetime constants](https://github.com/influxdata/ui/blob/master/src/utils/datetime/constants.ts) file. That way, it doesn't need to be defined in this file and redefined in the tests. If you don't move it to that file in this PR, at the very least I think this should be moved out of the function declaration and higher up into the file since it's a constant and it doesn't depend on any argument passed into the function. No need to redeclare this constant every time the function is called.